setup.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+"""chaostoolkit builder and installer"""
+
+import sys
+import io
+
+import setuptools
+
+sys.path.insert(0, ".")
+from chaosext import __version__
+sys.path.remove(".")
+
+name = 'chaostoolkit-ext'
+desc = 'Chaos Toolkit Extension Template'
+
+with io.open('README.md', encoding='utf-8') as strm:
+    long_desc = strm.read()
+
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',  
+    'Intended Audience :: Developers',
+    'License :: Freely Distributable',
+    'Operating System :: OS Independent',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: Implementation',
+    'Programming Language :: Python :: Implementation :: CPython'
+]
+author = 'chaostoolkit Team'
+author_email = 'contact@chaostoolkit.org'
+url = 'http://chaostoolkit.org'
+license = 'Apache License Version 2.0'
+packages = [
+    'chaosext'
+]
+
+needs_pytest = set(['pytest', 'test']).intersection(sys.argv)
+pytest_runner = ['pytest_runner'] if needs_pytest else []
+test_require = [
```

Comment:
wonder if we shouldn't have a `requirements-dev.txt` instead?

Reply:
Added.